### PR TITLE
54 - Change usermode.

### LINKED ISSUE #54: As a guardian I would like to be able to switch to citizen mode so that I can give the tablet to the citizen for use.
Tasks:
- [x] There should be an icon in the GirafAppBar that changes to citizen mode

### PR DESCRIPTION
**Pull request by User12837985123**
Closes #54

### CODE REVIEW

File: lib/screens/weekplan_screen.dart [hunk 1]
```diff
@@ -48,39 +53,126 @@ class WeekplanScreen extends StatelessWidget {
...
+                  key: const Key('visibilityMon'),
+                  visible: snapshot.data.item1 == 'Guardian' ||
+                           snapshot.data.item2 == 1,
+                  child: Expanded(
+                      child: Card(
+                          color: const Color(0xFF007700),
+                          child: _day('Mandag', tasksList))),
+                );
+              },
+            ),
```
Comment (LukasRH) [inline lib/screens/weekplan_screen.dart]: Could these Streambuilders maybe be made into a function to reduce code? 
As far as we can see, only three things changes in the code. (snapshot.data.item2, color,  _day)

File: lib/widgets/giraf_app_bar_widget.dart [hunk 2]
```diff
@@ -24,6 +28,10 @@ class GirafAppBar extends StatelessWidget implements PreferredSizeWidget {
   /// Contains the functionality of the toolbar.
   final ToolbarBloc toolbarBloc;
 
+  /// Contains the functionality of changing the weekplan.
+  final UserInfoBloc userInfoBloc;
+
+  //
```
Comment (LukasRH) [inline lib/widgets/giraf_app_bar_widget.dart]: Empty comment?

File: lib/bootstrap.dart [hunk 3]
```diff
@@ -4,6 +4,8 @@ import 'package:weekplanner/blocs/pictogram_bloc.dart';
 import 'package:weekplanner/blocs/pictogram_image_bloc.dart';
 import 'package:weekplanner/blocs/settings_bloc.dart';
 import 'package:weekplanner/blocs/toolbar_bloc.dart';
+import 'package:weekplanner/blocs/user_info_bloc.dart';
+import 'package:weekplanner/blocs/toolbar_bloc.dart';
```
Comment (LukasRH) [inline lib/bootstrap.dart]: Duplicate import of toolbar_bloc

File: pubspec.yaml [hunk 4]
```diff
@@ -11,6 +11,8 @@ dependencies:
   http: ^0.12.0+1
   shared_preferences: ^0.5.1+1
   injector: ^1.0.6
+  tuple: ^1.0.2
+  mockito: ^4.0.0
```
Comment (LukasRH) [inline pubspec.yaml:19]: mockito is also in dev dependency, which is unnecessary when in normal dependencies. Might be the reason why the build failed.

**Review by LukasRH — CHANGES_REQUESTED** [review]
@aau-giraf/sw613f19 review.
Looks fine, there is some inconsistency in whitespace in the files. Such as several empty lines between code, or empty lines in the bottom of the file, but above the last closing bracket.

**Review by LukasRH — APPROVED** [review]

**Comment by TobiasPalludan** [comment]
Closed due to conflict with #100

**Pull request closed without merging**